CHANGELOG.md

```diff
@@ -29,7 +29,22 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 - Changed from POST to PATCH calls for tokens & providers to eliminate entry
   duplication. (#10)
 
-- Added environment var for setting bind address and port. (#14)
+- Added config loading from YAML files using
+  `github.com/iver-wharf/wharf-core/pkg/config` together with new config models
+  for configuring wharf-provider-github. See `config.go` or the reference
+  documentation on the `Config` type for information on how to configure
+  wharf-provider-github. (#19)
+
+- Added config for setting bind address and port. (#14, #19)
+
+  - Environment variable: `WHARF_API_URL`
+  - YAML: `api.url`
```

Comment:
This doesn't seem right.
Shouldn't it be `WHARF_HTTP_BINDADDRESS` and `http.bindAddress`?

Reply:
Yea, probably should